index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header>
+   <nav class="top-header">
+     <a class="welcome identical-to" href="#">Welcome</a>
```

Comment:
- [ ] Great that you have wrapped the logo with `<a>` tag as per the suggestions from previous reviewer. However, you should remove the `text-decoration` style of the `<a>` tag, use the `font-size of 20px` with `font-weight of 600` as per the Figma design

Reply:
Thank you for your review, this changes you requested does not tally with Figma design. Here is the link of the template I am designing https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988 .
confirm that the font size is s 700px which is what I used and not 600 as you requested. Secondly, the terxt-decoration style removes the underline in the welcome link. If I remove it, then the welcome will be underlined but unfortunately it is not underlined in the Figma. Again the font size is 40px, although I used 48px, but I have changed it  40px and not to 20px as you requested. Hence, this changes is not necessary, thank you for your time. The paragraph is what is given the font-size of 20px and weight of 600px according to Figma in the above link

Reply:
Hi @stephnna :wave: 
Thank you for the feedback and let me take this chance to clarify. 
The text-decoration:- It is good that it has no underline now. There reason I asked for you to use the `text-decoration: none` is because I was seeing underline `Welcome` when I review it.
The font-size and font-weight: If you may take a look at the following snapshot taken form the Figma design you its link provided above. Please see the 'Welcome' text on the left colomun and the Typography on right column.
![font-size-font-weight](https://user-images.githubusercontent.com/44057863/164890731-9ab326ad-17d4-437e-ae2c-60ff1a9b3a2c.png)

thanks again for the feedback.

Reply:
- [ ] You should use a 20px font size in the logo. Please change it.
![image](https://user-images.githubusercontent.com/38635158/164892198-9434ae4e-2417-4194-ab47-2eaad9e0add0.png)
### ORIGINAL DESIGN
![image](https://user-images.githubusercontent.com/38635158/164892186-f633dc57-b8c5-421b-b4be-f24e1dc07a5f.png)


### OPTIONAL

- [ ] Also ıt was not mentioned before but you should move logo outside the nav tag. it will be like that. `<header><a></a><nav></nav></header>`